backend/core/src/listings/entities/listing.entity.ts

```diff
@@ -338,6 +340,16 @@ class Listing extends BaseEntity {
   @IsEnum(CountyCode, { groups: [ValidationsGroupsEnum.default] })
   @ApiProperty({ enum: CountyCode, enumName: "CountyCode" })
   countyCode: CountyCode
+
+  applicationCount: number
+
+  @AfterLoad()
```

Comment:
I don't think we need an extra method decorated with @AfterLoad here, see  how property.entity.ts has implemented `getUnitsSummarized`:
```
  @Expose()
  @ApiProperty({ type: UnitsSummarized })
  get unitsSummarized(): UnitsSummarized | undefined {
    if (Array.isArray(this.units) && this.units.length > 0) {
      return transformUnits(this.units)
    }
  }
```

Reply:
Cool, I'm good with using getters instead of AfterLoad.